packages/http-client-java/generator/http-client-generator-clientcore-test/package.json

```diff
@@ -12,26 +12,26 @@
     "spector-stop": "tsp-spector server stop"
   },
   "dependencies": {
-    "@typespec/spector": "0.1.0-alpha.12",
-    "@typespec/spec-api": "0.1.0-alpha.4",
-    "@typespec/http-specs": "0.1.0-alpha.20",
+    "@typespec/spector": "0.1.0-alpha.13",
+    "@typespec/spec-api": "0.1.0-alpha.6",
+    "@typespec/http-specs": "0.1.0-alpha.22",
     "@typespec/http-client-java": "file:../../typespec-http-client-java-0.3.2.tgz",
     "@typespec/http-client-java-tests": "file:"
   },
   "overrides": {
-    "@typespec/compiler": "^1.0.0-rc.1",
-    "@typespec/http": "^1.0.0-rc.1",
-    "@typespec/rest": "~0.69.0",
-    "@typespec/versioning": "~0.69.0",
-    "@typespec/openapi": "^1.0.0-rc.1",
-    "@typespec/xml": "~0.69.0",
-    "@typespec/events": "~0.69.0",
-    "@typespec/sse": "~0.69.0",
-    "@typespec/streams": "~0.69.0",
-    "@azure-tools/typespec-azure-core": "~0.55.0",
-    "@azure-tools/typespec-client-generator-core": "~0.55.1",
-    "@azure-tools/typespec-azure-resource-manager": "~0.55.0",
-    "@azure-tools/typespec-autorest": "~0.55.0"
+    "@typespec/compiler": "~1.0.0",
+    "@typespec/http": "~1.0.0",
+    "@typespec/rest": "~0.70.0",
+    "@typespec/versioning": "~0.70.0",
+    "@typespec/openapi": "~1.0.0",
+    "@typespec/xml": "~0.70.0",
+    "@typespec/events": "~0.70.0",
+    "@typespec/sse": "~0.70.0",
+    "@typespec/streams": "~0.70.0",
+    "@azure-tools/typespec-azure-core": "~0.56.0",
+    "@azure-tools/typespec-client-generator-core": "~0.56.0",
+    "@azure-tools/typespec-azure-resource-manager": "~0.56.0",
+    "@azure-tools/typespec-autorest": "~0.56.0"
```

Comment:
I am using the `~` for allow of patch only.

If we still see problem with versions, we can just fix it at e.g. `1.0.0`